### PR TITLE
features: mention new effect chain features

### PR DESCRIPTION
**Pull request by ronso0**
https://deploy-preview-327--mixxx-website.netlify.app/features/#effects

### CODE REVIEW

File: theme/templates/pages/features.html [hunk 1]
```diff
@@ -117,7 +117,7 @@ <h2><a class="headline-link" href="#samplers">{% trans %}Sampler Decks{% endtran
 <div id="effects" class="docs-section container">
   <h2><a class="headline-link" href="#effects">{% trans %}Effects Chains{% endtrans %}</a></h2>
   <p>
-    {% trans %}Link up to three effects in a chain for a unique twist on your mix. Fine tune your sound by tweaking each parameter individually. Customize your workflow by assigning different parameters to adjust with the effect's metaknob. Focus different effects from a controller for flexible control of effect chains, whether your controller has a full effects section or a single knob.{% endtrans %}
+    {% trans %}Link up to three effects in a chain for a unique twist on your mix. Fine tune your sound by tweaking each parameter individually. Customize your workflow by assigning different parameters to adjust with the effect's metaknob. Focus different effects from a controller for flexible control of effect chains, whether your controller has a full effects section or a single knob. Rearrange and hide parameters, and set defaults. Export your custom effect chains to share it with others.{% endtrans %}
```
Comment (Swiftb0y) [inline theme/templates/pages/features.html]: lets also mention the availability from the mixer, something like this?
```suggestion
    {% trans %}Link up to three effects in a chain for a unique twist on your mix. Fine tune your sound by tweaking each parameter individually. Customize your workflow by assigning different parameters to adjust with the effect's metaknob. Focus different effects from a controller for flexible control of effect chains, whether your controller has a full effects section or a single knob. Rearrange and hide parameters, and set defaults. Export your custom effect chains to share it with others. Assign your custom chains to the Filter/QuickEffect knob to add color right from your mixer controls.{% endtrans %}
```

Comment (ronso0) [reply theme/templates/pages/features.html]: Yup!

**Comment by ronso0** [comment]
Hmm, this is a long paragraph now, with a lot of imperatives.

**Comment by ywwg** [comment]
I think it's fine -- unless you have an edit you want to make I think we can merge it

**Comment by ronso0** [comment]
Go go go!

**Pull request merged**